src/app/Library/CrudPanel/Traits/Relationships.php

```diff
@@ -299,6 +299,10 @@ private static function getPivotFieldStructure($field)
         $pivotSelectorField['delay'] = 500;
         $pivotSelectorField['placeholder'] = trans('backpack::crud.select_entry');
         $pivotSelectorField['label'] = \Str::of($field['name'])->singular()->ucfirst();
+        $pivotSelectorField['validationRules'] = 'required';
+        $pivotSelectorField['validationMessages'] = [
+            'required' => 'The pivot field is required.',
```

Comment:
This should become a lang string, so people can translate it from English to their language.